dspy/primitives/template.py

```diff
@@ -1,14 +1,16 @@
-import regex
 import typing as t
-from dspy.signatures.signature import Signature
+
+import regex
+
 from dspy.primitives.example import Example
+from dspy.signatures.signature import Signature
 
 
 def passages_to_text(passages: t.Iterable[str]) -> str:
     assert len(passages) > 0
     if len(passages) > 1:
         return "\n".join(
-            [f"[{idx + 1}] <<{text}>>" for idx, text in enumerate(passages)]
+            [f"[{idx + 1}] <<{text}>>" for idx, text in enumerate(passages)],
         )
```

Comment:
I think stuff like this should ideally live inside the TypedPredictor, as a (default?) formatting for lists of string.
But I guess that can be a later commit.